test/oli/publishing/publisher_test.exs

```diff
@@ -0,0 +1,46 @@
+defmodule Oli.Publishing.PublisherTest do
+  use Oli.DataCase
+
+  import Oli.Factory
+
+  alias Oli.Publishing.Publisher
+  alias Oli.Repo
+
+  describe "changeset/2" do
+    test "changeset should be valid with correct attributes" do
+      changeset =
+        build(:publisher)
```

Comment:
nit: instead of piping one item in, it would stylistically be better not to use pipe here re: general elixir style guide. (applies to few other lines)